example/src/main/java/io/netty/example/tunechoserver/InetUtils.java

```diff
@@ -0,0 +1,397 @@
+/*
+ * Copyright 2012 The Netty Project
+ *
+ * The Netty Project licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package io.netty.example.tunechoserver;
+
+import java.net.InetAddress;
+
+import io.netty.buffer.ByteBuf;
+
+public final class InetUtils {
```

Comment:
would this class be useful outside the scope of examples? For example we have a `NetUtils` class which seems to be targeting similar things. Perhaps we could put this class in an `internal` package for now?